Simplenote.xcodeproj/project.pbxproj

```diff
@@ -3,7 +3,7 @@
 	archiveVersion = 1;
 	classes = {
 	};
-	objectVersion = 54;
+	objectVersion = 70;
```

Comment:
This requires a more recent Xcode version than the `16.1` one currently used by CI, so we might want to upgrade the `IMAGE_ID` too